api/client.go

```diff
@@ -223,3 +223,10 @@ func (c *Client) Delete(ctx context.Context, req *DeleteRequest) error {
 	}
 	return nil
 }
+
+func (c *Client) Heartbeat(ctx context.Context) error {
+	if err := c.do(ctx, http.MethodGet, "/", nil, nil); err != nil {
```

Comment:
If all you're doing is checking for the server, a `HEAD` request is more appropriate